InvenTree/templates/js/build.js

```diff
@@ -32,12 +32,17 @@ function newBuildOrder(options={}) {
 }
 
 
-function makeBuildOutputActionButtons(output, buildInfo) {
+function makeBuildOutputActionButtons(output, buildInfo, lines) {
     /* Generate action buttons for a build output.
      */
 
     var buildId = buildInfo.pk;
-    var outputId = output.pk;
+
+    if (output) {
+        outputId = output.pk;
+    } else {
+        outputId = 'untracked';
```

Comment:
Translation missing ;-)

Reply:
The 'untracked' text here is an internal variable for JS - it is not user facing :)